content/en/tracing/trace_pipeline/adaptive_sampling.md

```diff
@@ -0,0 +1,154 @@
+- Match your specified monthly budget.
+- Ensure visibility for low-traffic services and endpoints by capturing at least one trace for each combination of service, resource, and environment every 5 minutes.
+
+To configure services to use adaptive sampling, follow the instructions listed below.
+
+## Requirements
+
+- Datadog Agent [7.53.0][2] or higher.
+- [Remote Configuration][3]  enabled for your Agent.
+- `APM Remote Configuration Write` [permission][4].  
+   **Note**: If you don't have this permission, ask your Datadog admin to update your permissions from your organization settings.
+
+### Tracing library versions
+
+The following table lists minimum tracing library versions required for adaptive sampling:
+
+| Language    | Minimum version required |
+|-------------|--------------------------|
+| Java        | [v1.34.0][5]             |
+| Go          | [v1.68.0][6]             |
+| Python      | [v2.9.6][10]             |
+| Ruby        | [v2.0.0][11]             |
+| Node.js     | [v5.16.0][12]            |
+| .NET        | [v2.54.0][13]            |
+| C++/Proxies | [v0.2.2][14]             |
+| PHP         | [v1.4.0][17]             |
+
+## Configure the adaptive sampling target
+
+To get started with adaptive sampling, you first need to pick a target strategy setting:
+
+- **Set Budget by Number of APM Hosts**: Configure a budget that is proportional to your allotment and the number of services onboarded (for example, based on the number of APM hosts)
+- **Set Budget by Data Volume**: Configure a fixed target in gigabytes per month
+
+
+|          | Budget by Number of APM Hosts                                                                                                              | Budget by Data Volume                                                                 |
+|----------|--------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------|
+| **Pros** | Scales with the number of APM hosts and the number of services onboarded; only set it once                                                 | Ensures you never go over budget                                                      |
+| **Cons** | Not a good fit if you want to stay below a specific volume, as it may vary depending on the number of hosts reporting APM data to Datadog | You have to edit the budget every time you onboard a new service to adaptive sampling |
+
+To set the adaptive sampling monthly target:
+1. Navigate to the [Ingestion Control][18] page.
+2. Click **Manage Adaptive Sampling Target**
+  {{< img src="/tracing/guide/adaptive_sampling/adaptive_sampling_target_cta.png" alt="Call to action to set adaptive sampling target" style="width:100%;">}}
+3. Choose a target strategy for sampling:
```

Comment:
I find it difficult to follow when something ends with a colon, which makes it seem obvious that something is coming next, but then a heading comes next, which makes it seem like the content is at a dividing point. I would usually suggest indenting this content so it appears in line with the intro, but I don't think you can do that with a heading. So instead of headings, I wonder if putting these in expanders or tabs might help make it clearer that the following instructions are part of the step where you choose your target strategy?